k8s.go

```diff
@@ -126,6 +126,15 @@ func (k K8sClient) deleteSecret(namespace string, key string) error {
 	return k.c.Secrets(namespace).Delete(key, nil)
 }
 
+func (k K8sClient) deleteCertificate(c Certificate, namespace string) error {
+	log.Printf("About to delete certificate " + c.Metadata.Name + " / " + namespace)
```

Comment:
I would either use the printf formatting here or call `log.Print` instead here.

Reply:
Ah yes, forgot about that one as I did use it in the other log statements. Will change.